### DISCUSSION TITLE
Installing ani-cli on linux mint

### DISCUSSION THREAD

**Discussion by ellischung** [pystardust/ani-cli#1153] [Q&A] [answered]
This is probably a dumb question but I'm new to linux mint and was wondering how exactly do I install ani-cli and what are the steps?
I ran sudo apt install ani-cli but I get this on the terminal:

Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package ani-cli

**Comment by port19x** [comment, marked as answer]
No worries. You're getting this error because linux mint does not currently have ani-cli in their package repositories.

At this point your best bet is probably to install it manually [(docs)](https://github.com/pystardust/ani-cli#installing-from-source)

I recall that we had a PPA at some point which makes it easier to install ani-cli on ubuntu based distributions such as Linux Mint.
Since ani-cli is now in debian unstable we have discontinued that package

**Comment by irennene** [comment]
It's ok.
